src/frontend/src/optimizer/plan_node/batch_hop_window.rs

```diff
@@ -32,24 +31,25 @@ use crate::utils::ColIndexMappingRewriteExt;
 #[derive(Debug, Clone, PartialEq, Eq, Hash)]
 pub struct BatchHopWindow {
     pub base: PlanBase,
-    logical: LogicalHopWindow,
+    logical: generic::HopWindow<PlanRef>,
     window_start_exprs: Vec<ExprImpl>,
     window_end_exprs: Vec<ExprImpl>,
 }
 
 impl BatchHopWindow {
     pub fn new(
-        logical: LogicalHopWindow,
+        logical: generic::HopWindow<PlanRef>,
```

Comment:
maybe we need to change the name here later.

Reply:
To what?